### DISCUSSION TITLE
Tag docker image with its sha hash

### DISCUSSION THREAD

**Discussion by AlexeyRaga** [docker/build-push-action#1055] [Q&A]
Hi all,

Is it possible to have a docker image that is built to be tagged with its own sha hash?

**Comment by crazy-max** [comment]
> with its own sha hash?

If you mean git commit sha, then you can use the metadata-action: https://github.com/docker/metadata-action?tab=readme-ov-file#typesha

More info: https://docs.docker.com/build/ci/github-actions/manage-tags-labels/
